client/state/data-layer/wpcom/jetpack-onboarding/index.js

```diff
@@ -0,0 +1,59 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get, noop } from 'lodash';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { errorNotice } from 'state/notices/actions';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { JETPACK_ONBOARDING_SETTINGS_SAVE } from 'state/action-types';
+
+/**
+ * Dispatches a request to save particular onboarding settings on a site
+ *
+ * @param   {Object} action Redux action
+ * @returns {Object} Dispatched http action
+ */
+export const saveJetpackOnboardingSettings = ( { dispatch, getState }, action ) => {
+	const { settings, siteId } = action;
```

Comment:
You could destructure the `action` arg directly, i.e. on the line above.

Reply:
Oops, just noticed https://github.com/Automattic/wp-calypso/pull/20906/files#diff-2bc1b67d0ef2a5e01eb95b99c58e1c62R47